### ISSUE TITLE
Area checklists (called "Local checklist") require an area definition

### ISSUE THREAD

**Issue by Tasilee** [AtlasOfLivingAustralia/specieslist-webapp#138]
"Species checklists" are lists of taxa within a **defined area**. We do not currently enforce the definition of the **area** (which is currently called "Region" and shouldn't be) and we need to, as a "Species checklist" is of little value without an associated area.

There would be a fair chance that many areas would be named and defined spatially within the SP's  gazetteer, but we would have to allow for the use of the Spatial Portal's other methods to define an area: Most of the Add to Map | Area options (which I presume we could link to?).

**Comment by ansell** [comment]
Is lists.ala.org.au intended in the future to solely be a curated set of species lists suitable for reuse by others? 

Conversely, will it always allow the inclusion of single-use lists, which one may apply to various areas for exploratory purposes, rather than as a representative list of species actually present in any one area?

**Comment by Tasilee** [comment]
Fair comment @ansell. As Lists were my brainchild, I suppose I can be audacious. 

Lists can be public or private, but regardless, a "Species checklist" is of little value without the definition of the area it refers to. Sure, right now you could create a private 'Species checklist' without an area (region, grrr) but if you return to the list after a fair delay, it would be nice if the area was defined. In this case, it is like not having adequate metadata. Yes, you MAY be able to define area layer, but it is best done as an integral part of the (Species Check) List.

If the area is NOT important, then another List type should be used.

**Comment by ansell** [comment]
My bad, I didn't realise you meant a specific one of the various specific list types.

**Comment by Rita-C** [comment]
@Tasilee , define an area for "Species checklist", can this be achieved by making "Spatial bounds for data (WKT)" mandatory for this type of lists? This is the last field on the following screen:
https://lists-test.ala.org.au/speciesList/upload

**Comment by Tasilee** [comment]
Thanks @Rita-C , yes, exactly so. At some point when we have more resources for this (and Andre has raised the need for a serious hit on Lists just this morning with me), we would tap into the SPs services to define an area (Add to Map | Area) functions. For example, they could draw a polygon, or use a name in the gazetteer, or import a shapefile/wkt or KML) etc. For now, we could reference the SP if they didn't have a WKT on tap.

**Comment by Rita-C** [comment]
Updated issue title to ... rename to "Area checklist", ref @Tasilee comment in #120.

**Comment by Rita-C** [comment]
https://upsource.ala.org.au/specieslist-webapp/review/SW-CR-2

**Comment by nickdos** [comment]
I'm thinking this change is asking too much of regular users... in that forcing them to use the current UI to provide an area via the "Spatial bounds for data (WKT)" is UX hell. How is a regular user supposed to know how to do this? Even I would baulk at this request, as it would take me an extra 10 minutes fiddling in some GIS tool to get a valid WKT string. I'd say 99% of users of the list tool wouldn't know how to do this and then only 10% of the remaining 1% would even be bothered actually doing it. So in effect we're relegating "Local checklist" to never be used by actual users.

I'm OK if the SP does it, assuming its automatically generated. This issue is for the UI of lists tool to enforce the check and I don't see it resulting a viable UX without **a lot of extra work** (assisting user in defining the area). That work is not justified IMO and I'm suggesting this issue be closed as "won't fix".

**Comment by Tasilee** [comment]
I can understand your position (as a developer :), but it depends who "regular users" are. I'd say most users are using lists as a shorthand way of managing multiple species. That is, they are not 'species checklists', but at some point, we will certainly have a data campaign to capture the many species checklists that are available. Most parks, reserves and islands have them. Such lists are important as a snapshot in time. Note also that the Spatial Portal's Area report detects intersections of the area of interest with 'species checklist' areas. This is valuable.

My issue is that a 'species checklist' without a defined area is **not** a 'species checklist'. 

We have two reasonable options. We either 

- Remove the option from lists (with a note somewhere as to how species checklists can be generated via the Spatial Portal) or 
- Leave the list type there and mandate the definition of the area with the same text if they don't have a WKT on hand. I don't think lists should be validating WKTs for now.

My preference is for for the latter, as we DO have an educational role to play here as long as we document the why. That reminds me that the List types need to be in the vocabulary manager and ideally have rollovers or equivalent.

If a user really has a genuine 'species checklist' and they think "I don't have the area in WKT" or "it is an island" or similar, then it makes sense to say either "Go to the Spatial Portal's Add to map | Area and either define or import that area (and then come back or add the list from there...)" or if they really don't have a 'species checklist', then say "Select a more appropriate list type".

Given that a valid WKT can be generated through the Spatial Portal in many ways (e.g., draw, import, gazetteer etc), as can the 'species checklist' itself, this fits with https://github.com/AtlasOfLivingAustralia/spatial-hub/issues/263.

In short, it does no harm to enforce the area requirement, even if the Lists tool doesn't have the current flexibility to import/check the WKT, as long as we provide a way forward if they really do have an area. I'd be very surprised if enforcement of area generates flack, as anyone who really has a genuine 'species checklist' knows that an area needs to be associated with the list of species.
